Jenkinsfile

```diff
@@ -204,7 +204,7 @@ pipeline {
                     }
                     steps {
                         sh './scripts/setup-ci.sh'
-                        sh 'echo "y" | ./scripts/make-release $TAG_NAME update_docker'
+                        sh './scripts/make-release $TAG_NAME update_docker'
```

Comment:
Does this cause issues in jenkins? 🤔 
Removing it make it less safe for local runs, which will still eventually be needed.

Reply:
I think we should not remove them, if that doesn't break the CI env. 
If at some point the CI run fails and we need to execute locally, not requiring confirmation will result in stuff being pushed, which might be dangerous.